custom_components/bosch_alarm/__init__.py

```diff
@@ -40,6 +41,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hass.data[DOMAIN][entry.entry_id] = panel
 
     def setup():
+        # Some panels don't support retrieving a serial number.
+        # We still need some form of identifier, so fall back
+        # to the entry id.
+        if not panel.serial_number:
+            panel.serial_number = entry.entry_id
+
+        # Remove old devices using the panel model as an identifier
+        device_registry = dr.async_get(hass)
+        for device_entry in dr.async_entries_for_config_entry(
+            device_registry, entry.entry_id
+        ):
```

Comment:
something wonky is going on with the indentation here....

Reply:
how *should* this sort of thing be indented?

Reply:
Oh, playing with the python formatter I guess this is an acceptable way to format it...
It's the `):` on a separate line that throws me for a loop.
In this case, I think we can just put the whole for clause on the same line

Reply:
easy, looks like after the device registry changes it now isnt even that long a line anymore haha